<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>564</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Smolt</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../icons/smolt-icon-48.png</normaloff>../icons/smolt-icon-48.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>My Profile</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QTableWidget" name="defaultTableWidget">
          <column>
           <property name="text">
            <string>Label</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Info</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Detailed Info</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QTableWidget" name="detailedTableWidget">
          <column/>
          <column/>
          <column/>
          <column/>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>564</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Send"/>
    <addaction name="action_My_Smolt_Page"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_Privacy_Policy"/>
    <addaction name="action_About"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Send">
   <property name="text">
    <string>&amp;Send</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_My_Smolt_Page">
   <property name="text">
    <string>&amp;My Smolt Page</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="action_Privacy_Policy">
   <property name="text">
    <string>&amp;Privacy Policy</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
